src/pages/packer/plugins/index.tsx

```diff
@@ -42,8 +42,11 @@ async function getStaticProps(ctx) {
 	 */
 	if ('props' in staticProps) {
 		// Partial nav data is provided from base getStaticProps, in menuItems
-		const partialNavData =
+		const baseMenuItems =
 			staticProps.props.layoutProps.sidebarNavDataLevels[2].menuItems
+		// Remove the authored title element and "Overview" link, these
+		// would be redundant with the styled overview item that we add.
+		const partialNavData = [baseMenuItems[0], ...baseMenuItems.slice(3)]
```

Comment:
🙋🏻 Same question here, and also wondering if there is a way we can abstract these two instances of slicing since they're special cases and could easily become out of sync?

Reply:
Great point - took a shot at this with `addBrandedOverviewSidebarItem` 👍